prow/bump.sh

```diff
@@ -53,10 +49,19 @@ if ! ($SED --version 2>&1 | grep -q GNU); then
   exit 1
 fi
 
+TAC=tac
+if which gtac &>/dev/null; then
+  TAC=gtac
+fi
+if ! which "$TAC" &>/dev/null; then
+  echo "tac (reverse cat) required. If on OS X then brew install coreutils." >&2
```

Comment:
ultra-tiny-nitty-knit: `'brew install coreutils'`, it's nice to delineate the command